src/commands/whoami.rs

```diff
@@ -2,11 +2,17 @@ use crate::settings::global_user::GlobalUser;
 use crate::terminal::{emoji, message};
 
 pub fn whoami(user: &GlobalUser) {
-    let msg = format!(
-        "{} You are logged with the email '{}'.",
-        emoji::WAVING,
-        user.email
-    );
+    // Only returns email if present (using email + API key for auth).
+    match &user.email {
+        Some(email) => {
+            let msg = format!(
+                "{} You are logged with the email '{}'.",
+                emoji::WAVING,
+                email
+            );
 
-    message::info(&msg);
+            message::info(&msg);
+        }
+        None => (),
```

Comment:
if we dont want to do api call for this release we should print out the api token instead of nothing for this command

Reply:
The api token is very sensitive information and doesn't say anything about the user...